docs/src/Theory/Atmos/Microphysics_1M.md

```diff
@@ -0,0 +1,158 @@
+# Microphysics
```

Comment:
Should this live in CloudMocrophysics.jl?

Reply:
I think it's ok to keep here. It talks about what package we are using and how we are coupling it to our model state variables

Reply:
Meaning - I moved all the docs on how individual microphysics process rates are computed. But kept the docs on how those rates are coupled to total energy here.